### PR TITLE
First Pull Request

### PR DESCRIPTION
**Pull request by nicecodingtoo**
Created four files with defined classes, as of yet, only the turn_spec passes.

### CODE REVIEW

**Comment by nicecodingtoo** [comment]
Completed and passed tests for files: card, turn, and deck.

**Comment by nicecodingtoo** [comment]
Final pull request.

File: flashcard_runner.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  # #
+  # # category - (@round.percent_correct_by_category(category)).to_i
+  # # percentage.to_1
+  # #
+  # # puts "#{percentage.to_1}"
+  # # end
+  #
+
+
+  guess = gets.chomp
```
Comment (cjsim89) [inline flashcard_runner.rb:52]: I think you were working through the runner file when you turned this in - currently this does not work as the spec intends, though it does output all of this to the terminal. You'll want to set up variables and use your class instances to run the game.

File: lib/deck.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    @cards = cards
+  end
+
+  def count
+    @cards.count
+  end
+
+  def cards_in_categories(category)
+    cards_in_categories = []
+  @cards.each  do |card|
```
Comment (cjsim89) [inline lib/deck.rb:14]: Careful of your indents in this method; it's a little hard to read as is.

File: lib/round.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
+class Round
+  attr_reader :deck, :turns
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+  end
+
+  def current_card
+    deck.cards[turns.count]
```
Comment (cjsim89) [inline lib/round.rb:10]: Be careful when referencing instance variables that you use the `@` sign consistently.

File: lib/round.rb [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  end
+
+  def take_turn(guess)
+    new_turn = Turn.new(guess, current_card)
+    @turns << new_turn
+    new_turn
+  end
+
+   def number_correct
+     @turns.count do |turn|
```
Comment (cjsim89) [inline lib/round.rb:20]: Did you intend to use `.each` here instead of `.count`...?

File: lib/turn.rb [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    guess == card.answer
+  end
+
+#This method either returns "Correct!" or "Incorrect."
+#based on whether the guess was correct or not.
+  def feedback
+    return "Correct!"
+  end
+
+  def feedback2
```
Comment (cjsim89) [inline lib/turn.rb:23]: This method `feedback2` is not in the spec, and I think you were trying to have two different methods return separate feedback. However, both sets of feedback (both Correct and Incorrect) should be able to be returned from the same `feedback` method. 

What kind of syntax could you use to determine a course of action in code, based on a condition?

File: spec/card_spec.rb [hunk 6]
```diff
@@ -1,4 +1,4 @@
-require './lib/card'
+require './lib/card.rb'
```
Comment (cjsim89) [inline spec/card_spec.rb:1]: You can just use `./lib/card`, you should not need an `.rb`. in these require statements.

**Pull request closed without merging**